src/main/kotlin/calculator/StringAddCalculatorParser.kt

```diff
@@ -0,0 +1,27 @@
+package calculator
+
+object StringAddCalculatorParser {
+    private val DEFAULT_DELIMITER_REGEX = "[,|:]".toRegex()
+    private val CUSTOM_DELIMITER_REGEX: Regex = "//(.)\n(.*)".toRegex()
+
+    fun parse(text: String?): List<StringAddCalculatorElement> {
+        if (text.isNullOrBlank()) { return emptyList() }
+        if (isNumber(text)) { return listOf(StringAddCalculatorElement.of(text)) }
+
+        CUSTOM_DELIMITER_REGEX.find(text)?.let {
+            val (delimiter, numbers) = it.destructured
+            return numbers.split(delimiter).map { number -> StringAddCalculatorElement.of(number) }
```

Comment:
```suggestion
            return numbers.split(delimiter).map { StringAddCalculatorElement.of(it) }
```
numbers라는 list에 안에는 number이 들어가 있어보여서, it으로 표현해도 괜찮아보이네요~

Reply:
네 그렇네요! 적용해보겠습니다~!